source/class/qx/tool/cli/commands/Compile.js

```diff
@@ -789,9 +789,9 @@ Framework: v${await this.getQxVersion()} in ${await this.getQxPath()}`);
       }
 
       for (let filename of classFiles) {
-        if (this.argv.verbose) {
-          qx.tool.compiler.Console.info(`Processing ${filename} ...`);
-        }
+        // if (this.argv.verbose) {
```

Comment:
How about you just delete the lines altogether, instead of commenting them out?

Reply:
Whatever, I just saw such kind of these traces commented. I thought someone from us will need this for some debug actions.